packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/index.js

```diff
@@ -16,6 +16,7 @@ import { useContentTypeLayout } from '../../../../hooks';
 import { getTrad } from '../../../../utils';
 import FieldComponent from '../../../FieldComponent';
 import Rectangle from './Rectangle';
+import useMainValue from './utils/useMainValue';
```

Comment:
Apparently, we have a convention of putting hooks into a folder called `hooks`. Could you move that file there?

```suggestion
import useMainValue from './hooks/useMainValue';
```

Reply:
Yes, of course.